src/services/FsData.service.ts

```diff
@@ -1,4 +1,4 @@
-import { IHttpPromiseCallbackArg } from "@types/angular";
+import { IHttpResponse } from "@types/angular";
```

Comment:
`IHttpPromiseCallbackArg`is deprecated. I replaced it with the newer one `IHttpResponse `